src/luarocks/static_flags.lua

```diff
@@ -3,7 +3,80 @@ local static_flags = {}
 package.loaded["luarocks.static_flags"] = static_flags
 
 local util = require("luarocks.util")
+local fetch = require("luarocks.fetch")
+local path = require("luarocks.path")
+local search = require("luarocks.search")
 
 util.add_run_function(static_flags)
 
+static_flags.help_summary = "Returns all static libraries with flags required for the compiler's Lua flag."
+
+static_flags.help_arguments = "{<name> [<version>]}"
+static_flags.help = [[
+The argument may be the name of locally available module with newest version
+or defined version (optional), which has already built all static libraries.
+
+This command is useful for building an application with static libraries.
+
+Example:
+$ gcc -o myapp main.c -llua $(luarocks static-flags module_name)
+]]
+
+local function print_static_flags(collected_libs)
+   local full_string = ""
+   for _, value in pairs(collected_libs) do
+      full_string = value .. full_string
```

Comment:
Don't concatenate in a loop like this: built up a table and then concat it together with `table.concat`

Reply:
Infact, you already have it in table form, so just use `table.concat`